### DISCUSSION TITLE
API changes - Mirth 4.0.1

### DISCUSSION THREAD

**Discussion by CHATCHJ** [waytohealth/mirth_channel_exporter#6] [General]
Hello,

I am not sure if Mirth changed something with their API but it seem s that when mirth_channel_exporter tries to reach out to /api/channels/statuses it is timing out with an error. When I tried to go there manually I was unable to get a response.

Do you have any ideas on how I can fix this?

**Comment by CHATCHJ** [comment]
Oh I found the issue it turns out that the API call above does work but it works with https:// and mirth_channel_exporter is using http.

**Reply by mkopinsky** [reply]
Is there something in the exporter hardcoded to use http? I always pass an endpoint as an environment variable, and include https in the URL
